applications/desktop/scenes/desktop_scene_pinsetup.c

```diff
@@ -45,6 +44,7 @@ bool desktop_scene_pinsetup_on_event(void* context, SceneManagerEvent event) {
 
 void desktop_scene_pinsetup_on_exit(void* context) {
     Desktop* app = context;
+    SAVE_DESKTOP_SETTINGS(&app->settings);
```

Comment:
Whole thing of loading setting each time is ugly as hell. I've been expecting API from desktop to change this setting  and desktop_setting to use this API.

Reply:
Sounds reasonable.
Will fix in next review.